pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>com.hotels</groupId>
     <artifactId>hotels-oss-parent</artifactId>
-    <version>4.3.1</version>
+    <version>5.0.0</version>
```

Comment:
Please add this change to the changelog too as changing the parent can have side effects that we sometimes need to track down later.